src/main/java/org/folio/processor/RuleProcessor.java

```diff
@@ -170,12 +177,13 @@ private void applyTranslation(StringValue stringValue, ReferenceDataWrapper refe
     Translation translation = stringValue.getDataSource().getTranslation();
     if (translation != null) {
       String readValue = stringValue.getValue();
-      RecordInfo recordInfo = stringValue.getRecordInfo();
       try {
         TranslationFunction translationFunction = translationHolder.lookup(translation.getFunction());
         String translatedValue = translationFunction.apply(readValue, index, translation, referenceData, metadata);
         stringValue.setValue(translatedValue);
       } catch (Exception e) {
+        log.warn("Error while applying translation: function= {}; stringValue= {}; error msg= {}.", translation.getFunction(), stringValue, e.getMessage());
```

Comment:
The same log.error() should be use for exception handling.

Reply:
done